app.rb

```diff
@@ -56,17 +44,23 @@ def menu
   def validate_input(input)
     case input
     when 1
-      list_all_books
+      Listing.list('Book')
+      menu
     when 2
-      list_all_people
+      Listing.list('People')
+      menu
     when 3
-      create_a_person
+      Creator.create('Person')
+      menu
     when 4
-      create_book
+      Creator.create('Book')
+      menu
     when 5
-      create_rental
+      Creator.create('Rental')
+      menu
     when 6
-      list_rentals_by_person_id
+      Listing.list('Rentals')
+      menu
     when 7
       puts `clear`
       puts 'Thank you for using Quest Student Library. See you next time!'
```

Comment:
- [ ] Kindly make sure the `main.rb` is one that handles options (ask the user for the initial choice of options (or to quit).) As the requirements says :point_down:

#### Make sure that the role of` main.rb` is only to start their code and ask the user for the initial choice of options (or to quit).

Reply:
We have implemented the requested changes.